Core/NLua/Method/LuaMethodWrapper.cs

```diff
@@ -288,7 +289,8 @@ int Call (LuaState luaState)
 						else
 							_Translator.Push (luaState, _LastCalledMethod.cachedMethod.Invoke (targetObject, _LastCalledMethod.args));
 					}
-				} catch (TargetInvocationException e) {
+				} catch (TargetInvocationException e) {
+					if (_Translator.interpreter.use_traceback) e.GetBaseException().Data["Traceback"] = _Translator.interpreter.GetDebugTraceback() as string;
```

Comment:
Why are you casting to string if `GetDebugTraceback` already return a string?

Reply:
Oversight from a time when it returned an object. Unnecessary Indeed.